letsencrypt-nginx/letsencrypt_nginx/parser.py

```diff
@@ -448,10 +448,7 @@ def _parse_server(server):
     :rtype: dict
 
     """
-    parsed_server = {}
-    parsed_server['addrs'] = set()
-    parsed_server['ssl'] = False
-    parsed_server['names'] = set()
+    parsed_server = {'addrs': set(), 'ssl': False, 'names': set()}
```

Comment:
Maybe format this with one variable per line?  It's confusing to initialise a bunch of variables in one line, particularly if they aren't all identical.